src/sentry/static/sentry/app/views/settings/components/forms/rangeField.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import InputField from 'app/views/settings/components/forms/inputField';
+import RangeSlider from 'app/views/settings/components/forms/controls/rangeSlider';
+
+type DefaultProps = {
+  formatMessageValue?: false | Function;
```

Comment:
Can we be more specific than `Function`? We know the required signature of this function.

Reply:
I had it more defined previously but it conflicted with the incoming `formatMessageValue` as it comes from the generic `fieldFromConfig`, I'll take a look again though